backend/src/test/java/com/github/devbk84/backend/service/ProductServiceTest.java

```diff
@@ -53,4 +54,46 @@ void getProductsById(){
         assertEquals(expected, actual);
         verify(productRepo).findById("69");
     }
+
+    @Test
+    void deleteEntryByID(){
+        Product expectedProduct = new Product(
+                "8",
+                "Haferdrink",
+                "Test",
+                "orderFav",
+                new BigDecimal("1.50")
+        );
+
+        doNothing().when(productRepo).deleteById(isA(String.class));
```

Comment:
Spannend, wo hast du das gefunden? Das kann man machen, so ein verify würde aber eig auch schon ausreichen in unserem Fall (und tatsächlich macht die Aussage in diesem Fall nichts, weshalb ich sie tatsächlich eher entfernen würde)